src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -134,6 +134,15 @@
     (insert-statement-to-actor! tx input))
   (-update-actor! [_ tx input]
     (update-actor! tx input))
+  (-wipe-actor! [_ tx input]
+    (delete-actor-st2st input)
+    (delete-actor-st2activ input)
+    (delete-actor-attachments input)
+    (delete-actor-statements input)
+    (delete-actor-agent-profile input)
+    (delete-actor-state-document input)
+    (delete-actor-actor input))
```

Comment:
I noticed here that `tx` is not being passed to the `delete-actor-*` functions. This is a serious error that will ensure that your delete ops will not work.

Despite not having added tests, the CI is already warning you about this. The linting CI job is emitting warnings about it:
```
src/db/postgres/lrsql/postgres/record.clj:103:20: warning: unused binding tx
src/db/sqlite/lrsql/sqlite/record.clj:137:20: warning: unused binding tx
```
Which you can see [here](https://github.com/yetanalytics/lrsql/actions/runs/6002993352/job/16280465970?pr=330). This is why you never turn off warnings: they may be actual errors!

BTW, if you want to know why we have a CI linter, Cliff and I use VSCode [Calva](https://calva.io/) which comes with the linter, [clj-kondo](https://github.com/clj-kondo/clj-kondo), preinstalled. We added this CI job so that people who use other editors can see the warnings we see.

Reply:
(As a side note, the unit tests are failing, but for a completely different reason that I addressed in #331, so you can fix it by merging in the `main` branch.)